### ISSUE TITLE
Option Enabling Tell websites not to sell or share my data

### ISSUE THREAD

**Issue by FrittenLoverMarvin** [allo-/ffprofile#279]
What do you think about adding an option to enable the feature **Tell websites not to sell or share my data**?
I suppose it may be similar to **Send websites a "Do Not Track" request** which is already possible.

![Sell_or_Share](https://github.com/allo-/ffprofile/assets/106709454/87ddc43a-e6f0-4533-a7dc-f8ca5010314b)

Thanks for this awesome tool. 👍

**Comment by allo-** [comment]
Yes, it should be added. Many sites may ignore it, but maybe some respect it and it could have legal implications in the future.

Setting: `privacy.globalprivacycontrol.enabled`
Possibly also: `privacy.globalprivacycontrol.pbmode.enabled` (the default is already `true`)
